services/api-server/src/simcore_service_api_server/exceptions/handlers/_handlers_backend_errors.py

```diff
@@ -1,12 +1,32 @@
+import logging
+
+from common_library.error_codes import create_error_code
+from servicelib.logging_errors import create_troubleshootting_log_kwargs
+from servicelib.status_codes_utils import is_5xx_server_error
 from starlette.requests import Request
 from starlette.responses import JSONResponse
 
 from ...exceptions.backend_errors import BaseBackEndError
 from ._utils import create_error_json_response
 
+_logger = logging.getLogger(__name__)
+
 
 async def backend_error_handler(request: Request, exc: Exception) -> JSONResponse:
     assert request  # nosec
     assert isinstance(exc, BaseBackEndError)
-
-    return create_error_json_response(f"{exc}", status_code=exc.status_code)
+    user_error_msg = f"{exc}"
```

Comment:
NOTE: remember the issue in the middleware that we debug. Are we sure that this exception does not contain any secrets? Or said differently: the user error mesages need to be really under control